app/assets/javascripts/nested_related_items.js

```diff
@@ -44,6 +53,27 @@
       return link;
     },
 
+    toggleAll: function() {
+      var _this = this;
+
+      $('#toggleAll').on('click', function(){
+        var toggleLink  = $(this);
+        toggleLink.toggleClass('open');
+        toggleLink.text(($(this).text() == 'Collapse all') ? 'Expand all' : 'Collapse all');
+        var links =  $('.mods_display_nested_related_item');
```

Comment:
Similarly, if we pass `el` when this is called in `init` we can use the `listItems()` function (which uses this same selector which is stored in the `options` object).

Reply:
✅  Alright, I *think* I did the thing.